profile.html

```diff
@@ -0,0 +1,145 @@
+        >
+          <path
+            d="M24 4c-11.05 0-20 8.95-20 20s8.95 20 20 20 20-8.95 20-20-8.95-20-20-20zm0 6c3.31 0 6 2.69 6 6 0 3.32-2.69 6-6 6s-6-2.68-6-6c0-3.31 2.69-6 6-6zm0 28.4c-5.01 0-9.41-2.56-12-6.44.05-3.97 8.01-6.16 12-6.16s11.94 2.19 12 6.16c-2.59 3.88-6.99 6.44-12 6.44z"
+          />
+          <path d="M0 0h48v48h-48z" fill="none" />
+        </svg>
+        <span>Jane Doe</span>
+        <p>
+          ______________<br />
+          ______________<br />
+          ______________
+        </p>
+      </section>
+
+      <section>
+        <button class="button__profile">
+          <svg
+            role="img"
+            aria-label="Number of questions"
+            xmlns="http://www.w3.org/2000/svg"
+            height="1rem"
+            viewBox="0 0 512 512"
+          >
+            <!--! Font Awesome Free 6.4.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2023 Fonticons, Inc. -->
+            <path
+              fill="#083248"
+              d="M256 512A256 256 0 1 0 256 0a256 256 0 1 0 0 512zM169.8 165.3c7.9-22.3 29.1-37.3 52.8-37.3h58.3c34.9 0 63.1 28.3 63.1 63.1c0 22.6-12.1 43.5-31.7 54.8L280 264.4c-.2 13-10.9 23.6-24 23.6c-13.3 0-24-10.7-24-24V250.5c0-8.6 4.6-16.5 12.1-20.8l44.3-25.4c4.7-2.7 7.6-7.7 7.6-13.1c0-8.4-6.8-15.1-15.1-15.1H222.6c-3.4 0-6.4 2.1-7.5 5.3l-.4 1.2c-4.4 12.5-18.2 19-30.6 14.6s-19-18.2-14.6-30.6l.4-1.2zM224 352a32 32 0 1 1 64 0 32 32 0 1 1 -64 0z"
+            />
+          </svg>
+          23
+        </button>
+        <button class="button__profile">
+          <svg
+            role="img"
+            aria-label="Bookmarked questions"
+            xmlns="http://www.w3.org/2000/svg"
+            height="1rem"
+            viewBox="0 0 384 512"
+          >
+            <!--! Font Awesome Free 6.4.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2023 Fonticons, Inc. -->
+            <path
+              fill="#083248"
+              d="M0 48V487.7C0 501.1 10.9 512 24.3 512c5 0 9.9-1.5 14-4.4L192 400 345.7 507.6c4.1 2.9 9 4.4 14 4.4c13.4 0 24.3-10.9 24.3-24.3V48c0-26.5-21.5-48-48-48H48C21.5 0 0 21.5 0 48z"
+            />
+          </svg>
+          11
+        </button>
+      </section>
+      <section class="card card__profile">
+        <span>Dark Mode</span>
+        <svg
+          role="img"
+          aria-label="Switch to Darkmode"
+          xmlns="http://www.w3.org/2000/svg"
+          height="1rem"
+          viewBox="0 0 576 512"
+        >
+          <!--! Font Awesome Free 6.4.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2023 Fonticons, Inc. -->
+          <path
+            fill="#083248"
+            d="M192 64C86 64 0 150 0 256S86 448 192 448H384c106 0 192-86 192-192s-86-192-192-192H192zm192 96a96 96 0 1 1 0 192 96 96 0 1 1 0-192z"
+          />
+        </svg>
```

Comment:
As this will be user interaction, nesting the icon inside a button would be better.